dd-sdk-android-timber/src/main/kotlin/com/datadog/android/timber/DatadogTree.kt

```diff
@@ -13,9 +13,12 @@ import timber.log.Timber
  * An implementation of a [Timber.Tree], forwarding all logs to the provided [Logger].
  *
  * @param logger the logger to use with Timber.
+ * @param minLogPriority the minimum log priority to proceed with. If priority is below this one,
+ * then log entry will be rejected. Default is -1 (allow all).
  */
 class DatadogTree(
-    private val logger: Logger
+    private val logger: Logger,
+    private val minLogPriority: Int = -1
```

Comment:
Is this one really necessary? After all people can set the min priority when building the logger passed in parameter.

Reply:
Updated my PR, replaced it by a secondary constructor which will create a default logger with min priority set.